### PR TITLE
Tt pretrained

### PR DESCRIPTION
**Pull request by jonthegeek**
`gp` doesn't like some long code lines, but everything else passes. You might need this for `covr` to be fully happy (plus presumably we'll still have Windows-vs-non-Windows issues):

```
Sys.setenv(NOT_CRAN = 'true')
goodpractice::gp()
Sys.setenv(NOT_CRAN = "")
```

### CODE REVIEW

File: vignettes/basic_usage.Rmd [hunk 1]
```diff
@@ -62,22 +62,27 @@ changing soon).
 ```{r}
 # Set the number of tokens to pad/truncate the result to.
 n_tokens <- 20L
-n_inputs <- length(text_inputs)
-tokenized <- torchtransformers::tokenize_bert(text_inputs$segment1,
-                                              text_inputs$segment2,
-                                              n_tokens = n_tokens)
+tokenized <- torchtransformers::tokenize_bert(
```
Comment (jonathanbratt) [inline vignettes/basic_usage.Rmd:65]: I don't like just using the default here, even though they happen to be correct for the model used. Can we explicitly set the tokenizer/vocab, with a comment that we have to use the right vocab to be compatible with the uncased bert model?

**Comment by jonathanbratt** [comment]
I can't comment on the relevant part of the code since it wasn't changed for this PR, but I'd like to have a bit more discussion  of the overall process in the documentation for `bert_classification`. Maybe mention that:
 - the function makes a model using the specified pre-trained bert
 - that model is then fine-tuned on the provided data
 - a pre-trained model already has a tokenizer specified, which will also be used to tokenize the fine-tuning data.

I like that the function wraps up these details so the user doesn't have to worry about them each time, but I also want to make it easy to find out what's going on underneath.

**Review by jonathanbratt — CHANGES_REQUESTED** [review]
Mostly add more documentation

File: vignettes/basic_usage.Rmd (hunk 1 above)
Comment (jonthegeek) [reply vignettes/basic_usage.Rmd:65]: Ha, sure! You wrote that part, I just changed the indentation 🙃

Comment (jonathanbratt) [reply vignettes/basic_usage.Rmd:65]: I know. :) When I wrote it, there wasn't an option for specifying the tokenizer--it was hard-coded in {tt}. That realization is partly what led to this whole set of PRs. :-D

Comment (jonthegeek) [reply vignettes/basic_usage.Rmd:65]: I'll write a story about handling this a little better. In theory tidy_bert_output should be able to deal with this using the original, untokenized data (or we can create that data as a tt dataset, which will become tokenized in the model process).

Comment (jonathanbratt) [reply vignettes/basic_usage.Rmd:65]: or maybe it was just hard-coded in the dataset constructor? I don't remember anymore. :-D

File: R/bert_classification-fit.R [hunk 2]
```diff
@@ -31,20 +37,19 @@
 #'   * A __matrix__ with 1 factor column.
 #'   * A factor __vector__.
 #'
-#' @param valid_x Depending on the context:
-#'   * A number between 0 and 1, representing the fraction of data to use for
-#'     model validation.
-#'   * Predictors in the same format as `x`. These predictors will be used for
-#'     model validation.
-#'   * `NULL`, in which case no data will be used for model validation.
+#' @param valid_x Depending on the context: * A number between 0 and 1,
```
Comment (jonathanbratt) [inline R/bert_classification-fit.R:40]: whoops, somehow this got reformatted in this commit.

**Review by jonathanbratt — CHANGES_REQUESTED** [review]
looks like one of the lists got accidentally reformatted

File: R/bert_classification-fit.R (hunk 2 above)
Comment (jonthegeek) [reply R/bert_classification-fit.R]: Grr. Comment reflow doesn't understand markdown.

**Review by jonathanbratt — APPROVED** [review]

**Pull request merged**